build-scripts/runtime_lib.cmake

```diff
@@ -156,8 +156,8 @@ if (WAMR_BUILD_LIB_RATS EQUAL 1)
 endif ()
 
 if (WAMR_BUILD_SIMD EQUAL 1 AND WAMR_BUILD_FAST_INTERP EQUAL 1)
-    if (NOT (WAMR_BUILD_TARGET MATCHES "AARCH64.*" OR WAMR_BUILD_TARGET MATCHES "ARM.*"))
-        message(STATUS "SIMDe doesnt support platform " ${WAMR_BUILD_TARGET})
+    if (WAMR_BUILD_PLATFORM STREQUAL "windows")
+        message(STATUS "SIMDe doesnt support platform " ${WAMR_BUILD_PLATFORM})
```

Comment:
I wonder if that should be error instead of just a status? At that stage users set `WAMR_BUILD_SIMD` so they expect to have it; in case it's not supported, maybe we should notify the users more explicitly about it?